packages/MSLG/README.MD

```diff
@@ -0,0 +1,107 @@
+# mslg
```

Comment:
To stay consistent with other packages, add this at the top of the readme

> *NOTE: This is a **preview** tool and breaking changes may happen before it is released*

Reply:
fixed